app/models/user.rb

```diff
@@ -141,6 +141,10 @@ def current_student?
       student.any?
   end
 
+  def notepad
```

Comment:
I'm not sure if I like adding `User#notepad` when it only applies to supervisor-users. I would love User have no knowledge of `Note` whatsoever. Could you move the `find_or_create` to the `Note` class and update the Supervisor::NotesController#update accordingly?